yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
i think you may have `yarn`'d in root without realizing it. this file can go away :D

Reply:
Deleted